sdk/master_board_sdk/README.md

```diff
@@ -42,3 +42,29 @@ to run the example run:
 sudo ./bin/exec MY_INTERFACE
 ```
 where MY_INTERFACE is the name of the network interface used to connect to the master board.
+
+How to run the Python example
+--------
+
+* Clone the repository: `git clone https://github.com/open-dynamic-robot-initiative/master-board.git`
+
+* Get into the repository: `cd master-board`
+
+* Set the number of controlled drivers by opening `sdk/master_board_sdk/include/master_board_sdk/defines.h` and tuning the `N_SLAVES_CONTROLED` constant: `N_SLAVES_CONTROLED 4` if you are using 4 driver boards (1 per leg)
+
+* In `sdk/master_board_sdk/example/example.py`, at the start of the script, set the number of controled drivers to the value you put in `defines.h`: `N_SLAVES_CONTROLED = 4  # Current number of controled drivers`
+
+* Get in `sdk/master_board_sdk/`: `cd sdk/master_board_sdk/`
+
+* Create a build folder: `mkdir build`
+
+* Get into the folder: `cd build`
+
+* Turn on Python bindings by setting `BUILD_PYTHON_INTERFACE` to `ON` and set `CMAKE_BUILD_TYPE` to `RELEASE` with: `ccmake ..`
```

Comment:
It is also possible to use `cmake -DBUILD_PYTHON_INTERFACE=ON -DCMAKE_BUILD_TYPE=Release ..`
And if one wants to use Pthon 3: `-DPYTHON_EXECUTABLE=$(which python3)`